### PR TITLE
Update to version 16.8

### PR DESCRIPTION
**Pull request by nicoddemus**

### CODE REVIEW

**Comment by nicoddemus** [comment]
Hmm probably requiring a rerender, will do it later

**Comment by jakirkham** [comment]
Went ahead and added the re-rendering in PR ( https://github.com/conda-forge/packaging-feedstock/pull/3 ).

**Comment by nicoddemus** [comment]
Thanks! Rebased on your branch. 😁

**Comment by jakirkham** [comment]
Would you be interested in helping maintain?

**Comment by nicoddemus** [comment]
Sure! Also added me as maintainer then.

**Comment by nicoddemus** [comment]
All builds have passed. 😁

**Comment by jakirkham** [comment]
Maybe we can rebase on `master` to clean this up a bit. 😉

**Comment by nicoddemus** [comment]
You mean to squash e2c83e35d2226644d011585c01c3e3ccc7a5904a and 517a0609354466c48f9a044e4051a666f14e8001?

**Comment by nicoddemus** [comment]
Didn't realize that you had merged the re-render. Rebased now. 😁

File: recipe/meta.yaml [hunk 1]
```diff
@@ -33,8 +35,10 @@ test:
 about:
   home: https://github.com/pypa/packaging
   license: Apache 2.0 or BSD 2-Clause
+  license_file: LICENSE
```
Comment (jakirkham) [inline recipe/meta.yaml:38]: Would be good if we could include both license files, but that is not so easy actually. Let's leave it for later.

xref: https://github.com/conda/conda-build/issues/1247

**Comment by jakirkham** [comment]
Thanks @nicoddemus .

**Pull request merged**